diff --git a/build/release.mk b/build/release.mk
index faa027239..df40f0c68 100644
--- a/build/release.mk
+++ b/build/release.mk
@@ -99,7 +99,7 @@ package-prep:
 	@# Resource directories
 	mkdir -p $(DIST_PATH)/config
 	cp -L config/README.md $(DIST_PATH)/config
-	OUTPUT_CONFIG=$(PWD)/$(DIST_PATH)/config/config.json go run ./scripts/config_generator
+	OUTPUT_CONFIG=$(PWD)/$(DIST_PATH)/config/config.json GOOS= GOARCH= go run ./scripts/config_generator
 	cp -RL fonts $(DIST_PATH)
 	cp -RL templates $(DIST_PATH)
 	rm -rf $(DIST_PATH)/templates/*.mjml $(DIST_PATH)/templates/partials/
@@ -159,8 +159,8 @@ else
 	cp $(GOBIN)/$(CURRENT_PACKAGE_ARCH)/$(MM_BIN_NAME) $(DIST_PATH_GENERIC)/bin # from cross-compiled bin dir
 endif
 
-	#Download MMCTL for $(MMCTL_PLATFORM)
-	scripts/download_mmctl_release.sh $(MMCTL_PLATFORM) $(DIST_PATH_GENERIC)/bin
+	@#Copy cross-compiled MMCTL
+	cp "${HOME}/go/src/github.com/mattermost/mmctl/mmctl" $(DIST_PATH_GENERIC)/bin
 
 ifeq ("darwin_arm64","$(CURRENT_PACKAGE_ARCH)")
 	echo "No plugins yet for $(CURRENT_PACKAGE_ARCH) platform, skipping..."
@@ -194,17 +194,6 @@ else
 	done
 endif
 
-	@# Products
-
-	@if [ -d $(BUILD_BOARDS_DIR) ] ; then \
-		echo "Copying web app files for Boards product"; \
-		mkdir -p $(DIST_PATH_GENERIC)/client/products/boards; \
-		cp -R $(BUILD_BOARDS_DIR)/mattermost-plugin/webapp/dist/* $(DIST_PATH_GENERIC)/client/products/boards/; \
-	else \
-		echo "Unable to find files for Boards product. Please ensure that the Focalboard repository is checked out alongside the server and run 'make build-product' in it."; \
-		exit 1; \
-	fi
-
 package-osx-amd64: package-prep
 	DIST_PATH_GENERIC=$(DIST_PATH_OSX_AMD64) CURRENT_PACKAGE_ARCH=darwin_amd64 PLUGIN_ARCH=osx-amd64 MMCTL_PLATFORM="Darwin-x86_64" MM_BIN_NAME=mattermost $(MAKE) package-general
 	@# Package