diff --git a/build/release.mk b/build/release.mk
index 017746342..3c3e76bea 100644
--- a/build/release.mk
+++ b/build/release.mk
@@ -99,7 +99,7 @@ package-prep:
 	@# Resource directories
 	mkdir -p $(DIST_PATH)/config
 	cp -L config/README.md $(DIST_PATH)/config
-	OUTPUT_CONFIG=$(PWD)/$(DIST_PATH)/config/config.json go run ./scripts/config_generator
+	OUTPUT_CONFIG=$(PWD)/$(DIST_PATH)/config/config.json GOOS= GOARCH= go run ./scripts/config_generator
 	cp -RL fonts $(DIST_PATH)
 	cp -RL templates $(DIST_PATH)
 	rm -rf $(DIST_PATH)/templates/*.mjml $(DIST_PATH)/templates/partials/
@@ -159,13 +159,14 @@ else
 	cp $(GOBIN)/$(CURRENT_PACKAGE_ARCH)/$(MM_BIN_NAME) $(DIST_PATH_GENERIC)/bin # from cross-compiled bin dir
 endif
 
+	@#Copy cross-compiled MMCTL
+	cp "${HOME}/go/src/github.com/mattermost/mmctl/mmctl" $(DIST_PATH_GENERIC)/bin
+
 ifeq ("darwin_arm64","$(CURRENT_PACKAGE_ARCH)")
-	echo "No plugins or mmctl yet for $(CURRENT_PACKAGE_ARCH) platform, skipping..."
+	echo "No plugins yet for $(CURRENT_PACKAGE_ARCH) platform, skipping..."
 else ifeq ("linux_arm64","$(CURRENT_PACKAGE_ARCH)")
-	echo "No plugins or mmctl yet for $(CURRENT_PACKAGE_ARCH) platform, skipping..."
+	echo "No plugins yet for $(CURRENT_PACKAGE_ARCH) platform, skipping..."
 else
-	#Download MMCTL for $(MMCTL_PLATFORM)
-	scripts/download_mmctl_release.sh $(MMCTL_PLATFORM) $(DIST_PATH_GENERIC)/bin
 	@# Prepackage plugins
 	@for plugin_package in $(PLUGIN_PACKAGES) ; do \
 		ARCH=$(PLUGIN_ARCH); \
