diff --git a/build/release.mk b/build/release.mk
index 1530e75c6..befe5c9b7 100644
--- a/build/release.mk
+++ b/build/release.mk
@@ -76,7 +76,7 @@ package:
 	@# Resource directories
 	mkdir -p $(DIST_PATH)/config
 	cp -L config/README.md $(DIST_PATH)/config
-	OUTPUT_CONFIG=$(PWD)/$(DIST_PATH)/config/config.json go run ./scripts/config_generator
+	OUTPUT_CONFIG=$(PWD)/$(DIST_PATH)/config/config.json GOOS= GOARCH= go run ./scripts/config_generator
 	cp -RL fonts $(DIST_PATH)
 	cp -RL templates $(DIST_PATH)
 	rm -rf $(DIST_PATH)/templates/*.mjml $(DIST_PATH)/templates/partials/
@@ -98,7 +98,7 @@ package:
 	cp -RL $(BUILD_WEBAPP_DIR)/dist/* $(DIST_PATH)/client
 
 	@#Download MMCTL
-	scripts/download_mmctl_release.sh "" $(DIST_PATH)/bin
+	#scripts/download_mmctl_release.sh "" $(DIST_PATH)/bin
 
 	@# Help files
 ifeq ($(BUILD_ENTERPRISE_READY),true)
@@ -127,7 +127,7 @@ endif
 
 
 	@# ----- PLATFORM SPECIFIC -----
-
+_:
 	@# Make osx package
 	@# Copy binary
 ifeq ($(BUILDER_GOOS_GOARCH),"darwin_amd64")
@@ -197,7 +197,7 @@ endif
 	rm -f $(DIST_PATH)/bin/platform.exe
 	rm -f $(DIST_PATH)/bin/mmctl.exe
 	rm -f $(DIST_PATH)/prepackaged_plugins/*
-
+package-linux: package
 	@# Make linux package
 	@# Copy binary
 ifeq ($(BUILDER_GOOS_GOARCH),"linux_amd64")
@@ -208,7 +208,7 @@ else
 	cp $(GOBIN)/linux_amd64/platform $(DIST_PATH)/bin # from cross-compiled bin dir
 endif
 	#Download MMCTL for Linux
-	scripts/download_mmctl_release.sh "Linux" $(DIST_PATH)/bin
+	#scripts/download_mmctl_release.sh "Linux" $(DIST_PATH)/bin
 	@# Prepackage plugins
 	@for plugin_package in $(PLUGIN_PACKAGES) ; do \
 		ARCH="linux-amd64"; \
