diff --git a/Makefile b/Makefile
index 5eaee80..e0e13bf 100644
--- a/Makefile
+++ b/Makefile
@@ -54,7 +54,7 @@ GO_LINKER_FLAGS ?= -ldflags \
 				    -X github.com/mattermost/mattermost-server/model.BuildEnterpriseReady=$(BUILD_ENTERPRISE_READY)"
 
 # GOOS/GOARCH of the build host, used to determine whether we're cross-compiling or not
-BUILDER_GOOS_GOARCH="$(shell $(GO) env GOOS)_$(shell $(GO) env GOARCH)"
+BUILDER_GOOS_GOARCH="$(shell $(GO) env GOHOSTOS)_$(shell $(GO) env GOHOSTARCH)"
 
 PLATFORM_FILES="./main.go"
 
